source/server-selection/server-selection.rst

```diff
@@ -521,6 +527,24 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is a document that configures how the
+server will perform hedged reads. It consists of the following keys:
+
+- ``enabled`` (default true): This enables hedging
+- ``delay`` (default true): This enables staggered reads
+
+To explicitly enable hedging, an empty document must be sent to the server for
+the `hedge` option. This enables hedging with the default values specified
+above. To not enable hedging, the `hedge` option must be omitted entirely.
```

Comment:
Double back-ticks here, too.

Could we rephrase as "Hedging will be disabled if the `hedge` option is omitted"?